common-rest/src/main/java/org/uniprot/api/rest/output/context/MessageConverterContext.java

```diff
@@ -52,6 +53,7 @@ MessageConverterContext<T> asCopy() {
                 .suggestions(this.suggestions)
                 .warnings(this.warnings)
                 .isLargeDownload(this.isLargeDownload)
+                .subsequence(subsequence)
```

Comment:
Write test - 
1. by default subseq will be false
2. you can set it false as well if true